src/syscheckd/src/db/src/db.cpp

```diff
@@ -197,33 +215,37 @@ TXN_HANDLE fim_db_transaction_start(const char* table, result_callback_t row_cal
 
 FIMDBErrorCode fim_db_transaction_sync_row(TXN_HANDLE txn_handler, const fim_entry* entry)
 {
-    std::unique_ptr<DBItem> syncItem;
-    auto retval { FIMDB_OK };
+    auto retval { FIMDB_ERR };
 
-    if (entry->type == FIM_TYPE_FILE)
-    {
-        syncItem = std::make_unique<FileItem>(entry);
-    }
-    else
+    if (entry)
```

Comment:
We can avoid an indentation level
```suggestion
    if (entry)
    {
        return FIMDB_ERR;
     }
     std::unique_ptr<DBItem> syncItem;
```